ROADMAP.md

```diff
@@ -1,40 +1,64 @@
+* Support for Kubernetes 1.9+
+* New storage providers
+  * NFS operator and CRDs (backed by arbitrary PVs) [#1551](https://github.com/rook/rook/issues/1551)
+  * Cassandra design [#1910](https://github.com/rook/rook/issues/1910)
+* CockroachDB
+  * Secure deployment using certificates [#1809](https://github.com/rook/rook/issues/1809)
+  * Helm chart deployment [#1810](https://github.com/rook/rook/issues/1810)
+  * Run on arbitrary PVs [#919](https://github.com/rook/rook/issues/919)
+* Minio
+  * Helm chart deployment [#1814](https://github.com/rook/rook/issues/1814)
+  * Run on arbitrary PVs [#919](https://github.com/rook/rook/issues/919)
+* Ceph
+  * Update to Mimic [#1536](https://github.com/rook/rook/issues/1536)
+  * Automated upgrade support (initial) [#997](https://github.com/rook/rook/issues/997)
+  * Manage an existing Ceph cluster (basic) [#1868](https://github.com/rook/rook/issues/1868)
+  * OSDs
+    * Run on arbitrary PVs (local storage) as an alternative to host path [#796](https://github.com/rook/rook/issues/796) [#919](https://github.com/rook/rook/issues/919)
+    * Minimize or eliminate running with privileged containers (e.g., reduced access to /dev) [#1944](https://github.com/rook/rook/issues/1944)
+    * Disk management (adding, removing, and replacing disks) [#1435](https://github.com/rook/rook/issues/1435)
+    * ceph-volume provisioning [#1342](https://github.com/rook/rook/issues/1342)
+  * Mgr and plugins
+    * Placement group balancer support (enable the mgr module)
+  * File
+    * NFS Ganesha CRD [#1799](https://github.com/rook/rook/issues/1799)
+    * Dynamic Volume Provisioning for CephFS [#1125](https://github.com/rook/rook/issues/1125)
+  * Object
+    * Multi-site configuration [#1584](https://github.com/rook/rook/issues/1584)
+    * CRD for object store users [#1583](https://github.com/rook/rook/issues/1583)
 
 ## Rook 1.0
 
-* Declare numerous types as v1
-* Automated upgrades
+* Declare Ceph to be stable v1
+* Durability of state (local storage support, config can be regenerated) [#1011](https://github.com/rook/rook/issues/1011) [#592](https://github.com/rook/rook/issues/592)
+* Integration testing improvements
+  * Incorporate new environments [#1315](https://github.com/rook/rook/issues/1315)
+  * Incorporate more architectures [#1901](https://github.com/rook/rook/issues/1901)
+* Support for dynamic provisioning of new storage types
+  * Dynamic bucket provisioning [#1705](https://github.com/rook/rook/issues/1705)
+  * Dynamic database provisioning [1704](https://github.com/rook/rook/issues/1704)
+* New storage providers
+  * Cassandra cluster creation and horizontal scaling [#1910](https://github.com/rook/rook/issues/1910)
+  * Nexenta [#1532](https://github.com/rook/rook/issues/1532)
+* Ceph
+  * CSI plug-in (rook-ceph-agent uses ceph-fuse, nbd-rbd / tcmu runner) [#1385](https://github.com/rook/rook/issues/1385)
```

Comment:
Isn't `plug-in` normally written as `plugin`?

Reply:
they are both acceptable ways to spell it :) https://en.wikipedia.org/wiki/Plug-in_(computing)